<launch>
	<node name="filter_world"  pkg="object_recognition" type="filter_pointcloud" >
		<param name="keep_organized" value="false" />
		<param name="voxel_size" value="0.005" />
		<param name="keep_organized" value="false" />
		<remap from="/filter_world/input" to="/object_recognition/world/pointcloud/raw" />
		<remap from="/filter_world/output" to="/object_recognition/world/pointcloud/filtered" />
	</node>
</launch>
